services/Skype.json

```
@@ -0,0 +1,29 @@
+{
+    "name": "Skype",
+    "documents": {
+        "tos": {
+            "location": "https://www.skype.com/en/legal/ios/tos/",
+            "contentSelector": "div.row:nth-child(2) > div:nth-child(1)"
```

Selecting the first child is precisely the standard behavior of CSS selectors 🙂 

```suggestion
            "contentSelector": "div.row:nth-child(2) > div"
```